contracts/token/Minter.sol

```diff
@@ -95,14 +110,35 @@ contract Minter is Manager, IMinter {
      * @dev Set the reward token amounts for the round. Only callable by the RoundsManager
      */
     function setCurrentRewardTokens() external onlyRoundsManager whenSystemNotPaused returns (bool) {
+        setInflation();
+
         currentMintableTokens = mintedTokensPerRound();
         currentMintedTokens = 0;
-        currentRedistributableTokens = redistributableTokensPerRound();
+        currentRedistributableTokens = redistributionPool;
         currentRedistributedTokens = 0;
 
         return true;
     }
 
+    /*
+     * @dev Set inflation based upon the current bonding rate
+     */
+    function setInflation() internal {
+        uint256 currentBondingRate = (bondingManager().getTotalBonded() * PERC_DIVISOR) / livepeerToken().totalSupply();
+
+        uint256 newInflation = inflation;
```

Comment:
any reason to use the `newInflation` variable? Or could we eliminate it just be updating inflation itself in the if statement?